homeassistant/components/shelly/number.py

```diff
@@ -54,19 +68,41 @@ class BlockNumberDescription(BlockEntityDescription, NumberEntityDescription):
     ),
 }
 
+RPC_NUMBERS: Final = {
+    "thermostat": RpcNumberDescription(
+        key="thermostat",
+        sub_key="hysteresis",
+        icon="mdi:chart-bell-curve-cumulative",
+        name="Hysteresis",
```

Comment:
The device user interface uses the name `Temperature hysteresis`, I think we should keep this name.

Reply:
Good point, will change.